src-v2/styles/ChannelPreview/ChannelPreview-theme.scss

```diff
@@ -0,0 +1,54 @@
+@use '../utils';
+
+:root {
+  --str-chat__channel-preview-font-family: var(--str-chat__font-family);
+  --str-chat__channel-preview-border-radius: 0;
+}
+
+.str-chat__theme-light,
+.str-chat__theme-dark {
+  --str-chat__channel-preview-color: var(--str-chat__text-color);
+  --str-chat__channel-preview-background-color: transparent;
+  --str-chat__channel-preview-border-radius: 0;
+  --str-chat__channel-preview-border-block-start: none;
+  --str-chat__channel-preview-border-block-end: none;
+  --str-chat__channel-preview-border-inline-start: none;
+  --str-chat__channel-preview-border-inline-end: none;
+  --str-chat__channel-preview-active-background-color: var(--str-chat__surface-color);
+  --str-chat__channel-preview-hover-background-color: var(--str-chat__secondary-surface-color);
+  --str-chat__channel-preview-latest-message-inactive-color: var(
+    --str-chat__text-low-emphasis-color
+  );
+}
+
+.str-chat__channel-preview {
+  @include utils.component-layer-overrides('channel-preview');
+
+  &--active {
+    background-color: var(--str-chat__channel-preview-active-background-color);
+  }
+
+  &:hover {
+    background-color: var(--str-chat__channel-preview-hover-background-color);
+  }
+
+  .str-chat__channel-preview-messenger--name {
+    font-size: 1rem;
+    line-height: 1.25rem;
+    font-weight: 500;
+  }
+
+  .str-chat__channel-preview-messenger--last-message {
+    font-size: 0.875rem;
+    line-height: 1rem;
+    color: var(--str-chat__channel-preview-latest-message-inactive-color);
```

Comment:
`--str-chat__channel-preview-latest-message-inactive-color` suggests some sort of disabled element, maybe `secondary-color`?